src/test/java/LukeTest.java

```diff
@@ -0,0 +1,15 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class LukeTest {
+    @Test
+    public void dummyTest1(){
+        assertEquals(1, 1);
+    }
+
+    @Test
+    public void dummyTest2(){
```

Comment:
For each line 7 and 12, shouldn't there be a space between the closing parenthesis `)` and its proceeding brace `{`?

Reply:
Thanks, I'll add a whitespace.